src/service/company_report_vs_peers/company_report_vs_peers.py

```diff
@@ -118,35 +117,44 @@ def get_most_recent_metric_by_scenario(
     def get_company_financial_profile(
         self,
         company_id: str,
+        year: str,
     ) -> dict:
         try:
             financial_profile = dict()
             metrics = [
-                {"scenario": "Actuals", "metric": "Revenue", "alias": "annual_revenue"},
-                {"scenario": "Actuals", "metric": "Ebitda", "alias": "annual_ebitda"},
                 {
-                    "scenario": "Actuals",
+                    "scenario": f"Actuals-{year}",
+                    "metric": "Revenue",
+                    "alias": "annual_revenue",
+                },
+                {
+                    "scenario": f"Actuals-{year}",
+                    "metric": "Ebitda",
+                    "alias": "annual_ebitda",
+                },
+                {
+                    "scenario": f"Actuals-{year}",
                     "metric": "Rule of 40",
                     "alias": "anual_rule_of_40",
                 },
                 {
-                    "scenario": "Budgeted growth",
+                    "scenario": f"Budgeted growth-{year}",
                     "metric": "Revenue",
                     "alias": "current_revenue_growth",
                 },
                 {
-                    "scenario": "Budgeted margin",
+                    "scenario": f"Budgeted margin-{year}",
                     "metric": "Ebitda",
                     "alias": "current_ebitda_margin",
                 },
                 {
-                    "scenario": "Budgeted",
+                    "scenario": f"Budgeted-{year}",
```

Comment:
The scenario should change to Budget-{year} with the new changes in the Data Base